include/circt/Dialect/OM/OMAttributes.td

```diff
@@ -49,6 +49,10 @@ def OMSymbolRefAttr : AttrDef<OMDialect, "SymbolRef", [TypedAttrInterface]> {
 
   let extraClassDeclaration = [{
     mlir::Type getType();
+    // Get the SymbolRefAttr to the symbol represented by this operation.
+    static om::SymbolRefAttr get(mlir::Operation *op);
+    // Get the SymbolRefAttr to this symbol name.
+    static om::SymbolRefAttr get(mlir::StringAttr symName);
```

Comment:
I think `AttrBuilderWithInferredContext` is appropriate for these helpers.

Reply:
Thanks @uenoku for catching this. Completely missed the attribute  builders.